packer/scripts/firefox.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -eu
+
+cd /opt && \
+  curl -O https://download-installer.cdn.mozilla.net/pub/devedition/releases/81.0b2/linux-x86_64/en-US/firefox-81.0b2.tar.bz2 && \
+  tar -xf firefox-81.0b2.tar.bz2 && \
+  rm -rf firefox-81.0b2.tar.bz2
```

Comment:
Can we use an environment variable for the version?

Reply:
What do you mean by that?

Reply:
Like this:

```
FIREFOX_VERSION="81.0b2`

https://download-installer.cdn.mozilla.net/pub/devedition/releases/${FIREFOX_VERSION}/linux-x86_64/en-US/firefox-${FIREFOX_VERSION}.tar.bz2
```

Then we can easily increment to the new version by changing the variable.